tensorflow/tensorflow.bzl

```diff
@@ -1128,7 +1128,7 @@ def tf_gpu_cc_test(
         kernels = kernels,
         linkopts = linkopts,
         linkstatic = linkstatic,
-        tags = tags + ["manual"],
```

Comment:
I'm not sure this change here is something we'd actually want. Adding @gunan to double check

Reply:
This change resulted in multiple tests that were tagged `manual` to run, but fortunately only one of them resulted in a failure, on Windows, due to the code using `lrand48` which does not exist on Windows.